@uportal/content-renderer/README.md

```diff
@@ -46,17 +46,22 @@ compile 'org.webjars.npm:uportal__content-renderer:{version number goes here}'
 
 ```html
 <content-portlet-renderer
-  portlet-html-url="{portlet-url}"></content-portlet-renderer>
+  portlet-html-url="{portlet-url}"
+></content-portlet-renderer>
 ```
 
 ### uPortal home style 'widget' renderer
 
 ```html
 <content-widget-renderer
-  template="{widgetTemplate string}",
-  config="{widgetConfig object}",
-  type="{widgetType string}",
-  url="{widgetUrl string}",
+  template="{widgetTemplate string}"
+  ,
```

Comment:
are commas valid separators for html attributes?

Reply:
woops, no